core/src/main/scala/org/apache/spark/scheduler/DAGScheduler.scala

```diff
@@ -493,6 +523,15 @@ class DAGScheduler(
     }
 
     assert(partitions.size > 0)
+
+    /**
+     * Makes sure that getPartitions occurs before
+     * the job submitter sends a message into the DAGScheduler actor.
+     * Although getPartitions may be called in rdd.partitions.length before this.
+     */
+    val start = System.nanoTime
+    getParentPartitions(rdd)
+    logInfo("Get these partitions took %f s".format((System.nanoTime - start) / 1e9))
```

Comment:
I don't think that we need this timing information, definitely not at `logInfo` level.  I see why you kept it for benchmarking purposes, but I'd like to remove it before the final commit.